docs/source/services/timelock_service/installation.rst

```diff
@@ -50,3 +70,39 @@ Timelock Server Installation
               "healthy": true
           }
       }
+
+Add client(s) to Timelock
+-------------------------
+
```

Comment:
I don't think so, because they still need to do a different thing for each step, and they mustn't forget to back up!